src/main/java/com/krafit/financialcalcs/LoanCalculatorApplication/LoanCalculatorRestController.java

```diff
@@ -0,0 +1,29 @@
+package com.krafit.financialcalcs.LoanCalculatorApplication;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class LoanCalculatorRestController {
+
+    @PostMapping("/loanCalculator")
+    public ResponseEntity<LoanCalculatorHelper> getLoanDetails(@RequestBody LoanCalculator lc){
```

Comment:
change this to calculateExpectedEMIRelatedValues

Reply:
changed to calculateExpectedEMIRelatedValues